### ISSUE TITLE
RHEL8 Workstation Installation requirements

### ISSUE THREAD

**Issue by gilesknap** [dls-controls/dev-c7#13]
@aawdls 

We need a couple of things to add to the list for RHEL 8 installation:

- crun
- a couple of RHEL 7 library versions
  - libcrypto.so.10   
  - libssl.so.10

The libraries make dls-python3 work on RHEL8 and the python3 workflow should then be good to go.

**Comment by gilesknap** [comment]
@aawdls Do you think we should put VSCode on this list?

**Comment by aawdls** [comment]
Yes. There are the installations in /dls_sw/apps maintained by someone (unknown to me), but we need to decide whether we want to continue using these or a system package.

**Comment by gilesknap** [comment]
@MJGaughran re installing system packages. Some of us with pcXXXX machines can do that but those with wsXXXX cannot and the distribution seems random. For your policy to work we need to make sure people get yum rights.

Or am I misunderstanding you?
